### DISCUSSION TITLE
Github Profile To Show Mastodon

### DISCUSSION THREAD

**Discussion by isaacjohnson-wellsky** [community/community#45568] [Actions]
### Select Topic Area

Product Feedback

### Body

Today, we can set LinkedIn and Twitter links in our profile

![image](https://user-images.githubusercontent.com/97455831/215179157-5d5fc810-fd69-4b13-a500-23c66561f624.png)

I believe a large portion of us have moved to Mastodon.  Asking you to consider allowing a Mastodon link.

The format, in case not aware, is @$user@$mastodonsite ... this turns into https://$mastodonsite/@$user

e.g. for myself @ijohnson@noc.social == https://noc.social/@Ijohnson 

It does feel like the Twitter->Mastodon has crossed the breakpoint (like MySpace->Facebook, Digg->Reddit, etc). But i trust your judgement if you dont feel the time is right to make it first class.

**Comment by radar-blip** [comment]
Check settings

![socialaccts](https://user-images.githubusercontent.com/39196767/216235783-648e2b31-866c-4b2d-b4fe-e233a90695c7.png)

**Reply by idjohnson** [reply]
Thank you! I see links now![Screenshot_20230202_052359_Chrome.jpg](https://user-images.githubusercontent.com/6699477/216312206-67560abd-1200-4c09-9044-ad8d1d53970d.jpg)

**Reply by radar-blip** [reply]
No problem. 👍🏻

**Comment by ssbarnea** [comment]
A big thanks for adding support for mastodon links!

It should also be noted that at this moment Matrix links are not recognized as social links, even if matrix even has a IANA recognized URI (as opposed to mastodon).

There is already a thread at https://github.com/orgs/community/discussions/46165 that asks about adding matrix support, maybe for start it would be nice to even just recognize matrix and add a specific icon for these links.

**Comment by akirk** [comment]
Should this work for self-hosted Mastodon instances or does it only work for big, well-known ones?
